js/widgets/toolbar.js

```diff
@@ -40,7 +41,8 @@ define( [
 				leftbutton: leftbutton,
 				rightbutton: rightbutton
 			});
-			this.element.attr( "role", role === "header" ? "banner" : "contentinfo" ).addClass( "ui-" + role );
+			this.element.attr( "role", role === "header" ? "banner" : "contentinfo" );
```

Comment:
This is crazy not new but none the less crazy lets make this an option like it should be